### PR TITLE
Feature/co 1365 edit textbook integration test

### PR DESCRIPTION
**Pull request by MohammedAlawami**

### CODE REVIEW

File: src/test/integration/integration_tests.py [hunk 1]
```diff
@@ -61,19 +62,35 @@ def test_missing_param(self, param, valid_params):
...
+        (valid_year, valid_term) = term_id.split("-")
+        courses = utils.get_courses({"term_id": term_id, "id": subject})
+        # looping through each course in the term.
+        for course in courses.json():
+            valid_course = course["id"]
+            # if sections is empty loop agian to find a course
+            # with a valid section
+            if course["sections"]:
+                # include a random section in the response
+                random_section = random.randint(0, len(course["sections"]) - 1)
```
Comment (tsoliangwu0130) [inline src/test/integration/integration_tests.py]: That could work. Just wanna let you know you can also use [random.sample](https://docs.python.org/3.7/library/random.html#random.sample) to pick samples directly.

File: src/test/integration/integration_tests.py [hunk 2]
```diff
@@ -61,19 +62,35 @@ def test_missing_param(self, param, valid_params):
...
+                    "term": valid_term,
+                    "subject": subject,
+                    "courseNumber": valid_course,
+                    "section": valid_section
+                }
+                # if a valid term, course and section was found,
+                # exit the function with return. Otherwise loop again
+                return
+
+    # if this line was excuted that means there was nothing found in the loop.
```
Comment (ianbrown9475) [inline src/test/integration/integration_tests.py]: Minor typo

File: src/test/integration/integration_tests.py [hunk 3]
```diff
@@ -61,19 +62,35 @@ def test_missing_param(self, param, valid_params):
...
-    global book_params
-    book_params = {
-        "academicYear": valid_year,
-        "term": valid_term,
-        "subject": subject,
-        "courseNumber": valid_course,
-        "section": valid_section
-    }
+    # Nested for loop to loop through each term, and each course, to return
+    # the first valid term, course, and a random section ther is
```
Comment (ianbrown9475) [inline src/test/integration/integration_tests.py]: Minor typo

File: src/test/integration/integration_tests.py [hunk 4]
```diff
@@ -61,19 +62,35 @@ def test_missing_param(self, param, valid_params):
...
+    for term in terms.json():
+        term_id = term["id"]
+        (valid_year, valid_term) = term_id.split("-")
+        courses = utils.get_courses({"term_id": term_id, "id": subject})
+        # looping through each course in the term.
+        for course in courses.json():
+            valid_course = course["id"]
+            # if sections is empty loop agian to find a course
```
Comment (ianbrown9475) [inline src/test/integration/integration_tests.py]: Minor typo here

**Review by ianbrown9475 — CHANGES_REQUESTED** [review]

**Pull request merged**